lib/roo_on_rails/routemaster/lifecycle_events.rb

```diff
@@ -31,7 +30,8 @@ def publish_event(event, force_publish:)
           begin
             publisher.publish!(force_publish: force_publish)
           rescue => e
-            NewRelic::Agent.notice_error(e)
+            # SENTRY: To be migrated once we're on sentry
+            Rails.logger.error("#{e.inspect} rescued in #{self.class.name}::publish_event")
```

Comment:
this unfortunately won't get surfaced anywhere meaningful — can we do something like `Raven.report_exception(e) if defined?(Raven)` or something, to make sure we're not losing these?